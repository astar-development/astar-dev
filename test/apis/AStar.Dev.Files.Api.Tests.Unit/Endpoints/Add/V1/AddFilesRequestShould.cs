using AStar.Dev.Utilities;
using JetBrains.Annotations;

namespace AStar.Dev.Files.Api.Endpoints.Add.V1;

[TestSubject(typeof(AddFilesRequest))]
public class AddFilesRequestShould
{
    [Fact]
    public void ContainTheAddTheExpectedProperties()
        => new AddFilesRequest { FilesToAdd = new List<FileDetailToAdd> { new() } }.ToJson().ShouldMatchApproved();
}
