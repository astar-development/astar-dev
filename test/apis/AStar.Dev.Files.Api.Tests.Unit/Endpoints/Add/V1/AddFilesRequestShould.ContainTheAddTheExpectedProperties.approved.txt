{
  "filesToAdd": [
    {
      "fileClassifications": [],
      "id": 0,
      "imageDetails": {
        "width": null,
        "height": null
      },
      "fileName": "",
      "directoryName": "",
      "fileSize": 0,
      "fileCreated": "0001-01-01T00:00:00+00:00",
      "fileLastModified": "0001-01-01T00:00:00+00:00"
    }
  ],
  "name": "Add Files",
  "httpMethod": "POST"
}